### PR TITLE
New designs

### PR DESCRIPTION
**Pull request by dgore7**
Implement latest figma designs for nine ball.
fancy stuff ✨

### CODE REVIEW

File: src/lib/common/ball.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
+export type BallColorSpec = {
+	label: string;
+	border: string;
+	stroke: string;
+	gradient: {
+		stops: string[];
```
Comment (oodtoon) [inline src/lib/common/ball.ts:6]: stops[0] is the only one that's used. We should remove it as an array if unused

File: src/lib/components/ControlPad.svelte [hunk 2]
```diff
@@ -1,10 +1,11 @@
 <script lang="ts">
-	import Ball, { type BallModel } from './Ball.svelte';
+	import Ball from './Ball.svelte';
 	import ControlButtons from './ControlButtons.svelte';
 	import { createEventDispatcher } from 'svelte';
 	import type { NineBallGame } from '$lib';
-	import SkullyIcon from './icons/SkullyIcon.svelte';
+	import deadBallSvg from '$lib/assets/balls/dead/1.svg';
```
Comment (oodtoon) [inline src/lib/components/ControlPad.svelte:6]: import other dead balls as well

File: src/lib/components/icons/PauseIcon.svelte
Comment (oodtoon) [inline src/lib/components/icons/PauseIcon.svelte:1]: Rename file since it makes no effing sense.

**Review by oodtoon — APPROVED** [review]

File: src/lib/components/ControlPad.svelte (hunk 2 above)
Comment (dgore7) [reply src/lib/components/ControlPad.svelte:6]: not available right now

**Pull request merged**